### ISSUE TITLE
MissingField in PredictiveMaintenanceSettings

### ISSUE THREAD

**Issue by OBerghmans** [skodaconnect/homeassistant-myskoda#34]
I have a similar issue to the existing #33 but another missing field. I might be able to add the phone in my profile but logging the ticket just as it might be an easy fix.

```
Logger: myskoda.rest_api
Source: /usr/local/lib/python3.12/site-packages/myskoda/rest_api.py:43
First occurred: 10:46:22 PM (8 occurrences)
Last logged: 10:48:07 PM

Failed to load data from url /v3/vehicle-maintenance/vehicles/XXXXXXXX. Return value was '{"maintenanceReport":{"capturedAt":"2024-09-25T09:42:38.949Z","inspectionDueInDays":715,"mileageInKm":665},"preferredServicePartner":{"name":"AAAAA","brand":"C","partnerNumber":"01234","id":"BELC01234","contact":{"phone":"+32xxxxxxx","url":""},"address":{"street":"xxxxxxxxxxxxxx 231","zipCode":"9876","city":"xxxxxxxxx","countryCode":"BEL"},"location":{"latitude":1.23,"longitude":5.67},"openingHours":[{"periodStart":"MONDAY","periodEnd":"SUNDAY","openingTimes":[{"from":"00:00:00","to":"23:59:00"}]}]},"predictiveMaintenance":{"setting":{"serviceActivated":true,"email":"mail@something.com"}}}'
Traceback (most recent call last):
  File "<string>", line 8, in __mashumaro_from_dict_json__
  File "<string>", line 9, in __mashumaro_from_dict_json__
mashumaro.exceptions.MissingField: Field "phone" of type str is missing in PredictiveMaintenanceSettings instance

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<string>", line 17, in __mashumaro_from_json__
  File "<string>", line 10, in __mashumaro_from_dict_json__
mashumaro.exceptions.InvalidFieldValue: Field "setting" of type PredictiveMaintenanceSettings in PredictiveMaintenance has invalid value {'serviceActivated': True, 'email': 'mail@something.com'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.12/site-packages/myskoda/rest_api.py", line 41, in _make_get_request
    data = deserialize(response_text)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<string>", line 19, in __mashumaro_from_json__
mashumaro.exceptions.InvalidFieldValue: Field "predictive_maintenance" of type Optional[PredictiveMaintenance] in Maintenance has invalid value {'setting': {'serviceActivated': True, 'email': 'mail@something.com'}}
```

**Comment by dvx76** [comment]
Should be fixed by skodaconnect/myskoda#40

**Comment by WebSpider** [comment]
This has been fixed in v1.4.2
